libraries/loader/src/loader.js

```diff
@@ -102,8 +102,8 @@ function loader(componentDirectory, virtualComponents = {}) {
   componentDirectory['dump-dot'] = {
     setup: () => console.log(renderGraph(componentDirectory, topoSorted)),
   };
-
-  return function(target, options = {}) {
+  // Implementing the closure property to stablish load.crashOnError() functionality
+  let myLoader = function(target, options = {}) {
```

Comment:
We typically don't name variables in "production" code starting with `my` -- it's common in examples and demos as a way of saying "you can name this whatever you want".  Let's call this `load`, since that's what it will be called by services that are using the library.

To avoid confusion, you can rename the `load` sub-function below (around line 134) to something like `recursiveLoad`.

Reply:
Yeah ! I will do the needful ! Thank You